python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_network_services.schema.yml

```
@@ -548,6 +548,58 @@ $defs:
                       route_map:
                         type: str
                         description: Route-map name.
+                  authentication:
+                    type: str
+                    valid_values:
+                      - simple
+                      - message-digest
+                    description: |-
+                      Configure OSPF authentication at the VRF level.
```

This is not strictly true. We are configuring it per interface still.
```suggestion
                      Configure OSPF authentication for all interfaces under the VRF.
```